chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -597,7 +610,19 @@ impl PeerManagerActor {
                     .entered();
 
                     match response {
-                        Ok(routing::actor::Response::AddVerifiedEdgesResponse(filtered_edges)) => {
+                        Ok(routing::actor::Response::AddVerifiedEdgesResponse(
+                            mut filtered_edges,
+                        )) => {
+                            // Don't send tombstones during the initial time.
+                            // Most of the network is created during this time, which results
+                            // in us sending a lot of tombstones to peers.
+                            // Later, the amount of new edges is a lot smaller.
+                            if let Some(skip_tombstones_until) = skip_tombstones_until {
+                                if act.clock.now() < skip_tombstones_until {
+                                    filtered_edges.retain(|edge| edge.removal_info().is_none());
```

Comment:
I think more conventional here would be edge.edge_type() == Removed

Reply:
good idea -done.